frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -47,12 +83,28 @@ export const DocumentDetailContainer: React.FunctionComponent<IDocumentDetailCon
       result && props.onUpdateSuccess();
     }
   };
-  return (
-    <DocumentDetailView
-      document={document}
-      isLoading={retrieveDocumentMetadataLoading || updateDocumentLoading}
-      onUpdate={onUpdateDocument}
-      onCancel={props.onCancel}
-    />
-  );
+
+  const onCancelUpdate = () => {
+    setIsEditable(false);
+  };
+
+  if (!isEditable) {
```

Comment:
nit: if(truthy) else
vs
if(falsey) else

Reply:
a yes, the javascript